### PR TITLE
docs: fix docker-env-setup.md

### PR DESCRIPTION
**Pull request by peng-yq**
## Relevant Issue (if applicable)

#1633
 
## Details

Just fix the manual.

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: docs/docker-env-setup.md [hunk 1]
```diff
@@ -12,24 +12,25 @@ sudo cp ./* /usr/bin/
...
 
-wget -O /etc/nydus/nydusd-config.fusedev.json https://raw.githubusercontent.com/containerd/nydus-snapshotter/"$TAG"/misc/snapshotter/nydusd-config.fusedev.json
-wget -O /etc/nydus/config.toml https://raw.githubusercontent.com/containerd/nydus-snapshotter/"$TAG"/misc/snapshotter/config.toml
+sudo wget -O /etc/nydus/nydusd-config.fusedev.json https://raw.githubusercontent.com/containerd/nydus-snapshotter/"$TAG"/misc/snapshotter/nydusd-config.fusedev.json
+sudo wget -O /etc/nydus/config.toml https://raw.githubusercontent.com/containerd/nydus-snapshotter/"$TAG"/misc/snapshotter/config.toml
 ```
 
-3. Download nydus image service release tarball from [the release page](https://github.com/dragonflyoss/nydus/releases). 
-```
+3. Download nydus image service release tarball from [the release page](https://github.com/dragonflyoss/image-service/releases).
```
Comment (imeoer) [inline docs/docker-env-setup.md:26]: `image-service` -> `nydus` :)

File: docs/docker-env-setup.md [hunk 2]
```diff
@@ -12,24 +12,25 @@ sudo cp ./* /usr/bin/
...
 ```
 
-3. Download nydus image service release tarball from [the release page](https://github.com/dragonflyoss/nydus/releases). 
-```
+3. Download nydus image service release tarball from [the release page](https://github.com/dragonflyoss/image-service/releases). 
+```shell
 # Get the latest version. If this version does not work for you, you can try v2.1.4
 TAG=`curl -s https://api.github.com/repos/dragonflyoss/nydus/releases/latest | grep tag_name | cut -f4 -d "\""`
-wget https://github.com/dragonflyoss/nydus/releases/download/"$TAG"/nydus-static-"$TAG"-linux-amd64.tgz
+wget https://github.com/dragonflyoss/image-service/releases/download/"$TAG"/nydus-static-"$TAG"-linux-amd64.tgz
```
Comment (imeoer) [inline docs/docker-env-setup.md:30]: Ditto.

File: docs/docker-env-setup.md (hunk 1 above)
Comment (peng-yq) [reply docs/docker-env-setup.md:26]: The PR submitted to the stable/v2.2 branch is also related to image-service, not nydus. When searching for https://github.com/dragonflyoss/image-service, it redirects to https://github.com/dragonflyoss/nydus. Should it be changed to nydus then? And what about the stable/v2.2 branch?

**Review by imeoer — APPROVED** [review]
Thanks, let's merge it first.

**Pull request merged**